examples/src/Auction/Offchain.hs

```diff
@@ -60,7 +60,7 @@ txBid ::
   m ()
 txBid p bid = do
   bidder <- ownPaymentPubKeyHash
-  [utxo] <- scriptUtxosSuchThat (A.auctionValidator p) (\_ _ -> True)
+  [(utxo, datum)] <- scriptUtxosSuchThat (A.auctionValidator p) (\_ _ -> True)
```

Comment:
IIUC endpoints could avoid using `scriptUtxosSuchThat` and instead use the new `spOutsFromCardanoTx`. If the later style is to be preferred I would expect the examples to be updated eventually.